Sources/Apollo/DataLoader.swift

```diff
@@ -1,52 +1,42 @@
 import Dispatch
```

Comment:
Probably? I remember there was someone against importing `Foundation` because they wanted it to work on Linux. Not sure if that is still the case, and we probably don't really care. Should every file just import `Foundation`?

Reply:
There's definitely foundation for Linux now - I don't think every file needs to import Foundation (some are just using pure Swift from the stdlib) but I do think we should prefer importing `Foundation` over this.

Reply:
see #1534 for some initial stabs at Linux from a contributor - looks like it's no longer as simple as just `import Foundation` 😭